sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/Utility.java

```diff
@@ -439,7 +440,7 @@ public static Response<DetectLanguageResultCollection> toDetectLanguageResultCol
                 )));
         }
         // Document errors
-        for (DocumentError documentError : languageResult.getErrors()) {
+        for (InputError documentError : languageResult.getErrors()) {
```

Comment:
did this come from generation or is this hand-authored? I'm confused because it's under /implementation

Reply:
It comes from generated code.

Reply:
Ah I see they changed the name